### ISSUE TITLE
Add ChromaDB knowledge base

### ISSUE THREAD

**Issue by ashpreetbedi** [phidatahq/phidata#60]

**Comment by hoxsec** [comment]
Hi guys, 

Tried implementing this feature, however I'm getting stuck at the following mypy test: 

### error: Function "chromadb.Client" is not valid as a type  [valid-type]

```python
@property
    def client(self) -> chromadb.Client:
        if self._client is None:
            return chromadb.Client(
                namespace=self.namespace,
                hostname=self.hostname,
                port=self.port,
                headers=self.headers,
                ssl=self.ssl,
            )
        return self._client
```

**Comment by ysolanky** [comment]
@hoxsec Thanks for contributing! Why don't you create a PR so that I can take a look at this

**Comment by hoxsec** [comment]
Thanks @ysolanky, PR created. https://github.com/phidatahq/phidata/pull/196

**Comment by ysolanky** [comment]
ChromaDb was released as a part of v2.4.27 :)
